libs/ha/ha_common_libs_k8s.py

```diff
@@ -711,7 +712,7 @@ def create_bucket_copy_obj(event, s3_test_obj=None, bucket_name=None, object_nam
                         event_clear_flg = False
                         continue
                     failed_bkts.append(bkt_name)
-                LOGGER.info("Failed to copy object to bucket %s", bkt_name)
+                LOGGER.exception("Failed to copy object to bucket %s", bkt_name)
```

Comment:
No need to print traceback again. This can be LOGGER.info only as traceback will be printed at line 705.